From 22df667e7bc76284750c462acc27b84fd7c72ad6 Mon Sep 17 00:00:00 2001
From: Binh Nguyen <binh.nguyen.uw@rvc.renesas.com>
Date: Fri, 31 Jul 2015 11:39:34 +0700
Subject: [PATCH 2/4] qtmultimedia: increase timeout in waitForAsyncEvent
 waitForAsyncEvent meet timeout may make error, video
 stop with out notification increase timeout from 300ms
 to 1000ms

Signed-off-by: Binh Nguyen <binh.nguyen.uw@rvc.renesas.com>
---
 .../src/gsttools/qgstvideorenderersink.cpp         |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gsttools/qgstvideorenderersink.cpp b/src/gsttools/qgstvideorenderersink.cpp
index 31ac94e..bb0b518 100644
--- a/src/gsttools/qgstvideorenderersink.cpp
+++ b/src/gsttools/qgstvideorenderersink.cpp
@@ -240,7 +240,7 @@ GstFlowReturn QVideoSurfaceGstDelegate::render(GstBuffer *buffer)
     m_renderReturn = GST_FLOW_OK;
     m_renderBuffer = buffer;
 
-    GstFlowReturn flowReturn = waitForAsyncEvent(&locker, &m_renderCondition, 300)
+    GstFlowReturn flowReturn = waitForAsyncEvent(&locker, &m_renderCondition, 1000)
                                                 ? m_renderReturn
                                                 : GST_FLOW_ERROR;
 
-- 
1.7.9.5

